### PR TITLE
Switch to evaldown for examples.

### PR DESCRIPTION
**Pull request by alexjeffburke**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "lint": "eslint . && prettier --check '**/*.{js,md}'",
     "test": "npm run test-node && npm run test-docs",
     "test-browser": "npm run build && npm run bundle && npm run build-tests && npm run bundle-tests && karma start --single-run --browsers ${BROWSER:-ChromeHeadless}",
-    "test-docs": "mocha `find documentation -name '*.md'`",
+    "test-docs": "mocha --require unexpected-markdown `find documentation -name '*.md'`",
```
Comment (papandreou) [inline package.json:20]: Shouldn't this use `evaldown --validate`?

Comment (alexjeffburke) [reply package.json:20]: Not yet - I’ve not gotten per snippet output implemented yet which means swapping would be a regression. But should be a breeze to add later :)

Comment (papandreou) [reply package.json:20]: Ah, okay :+1:

Comment (alexjeffburke) [reply package.json:20]: ...to clarify, what I mean is at the level of an individual output line via the spec reporter. But the stringified error report is obviously present, so if we can live with that we could switch now and it’ll get better at some future release.

Comment (papandreou) [reply package.json:20]: I'm happy to make the switch now given that the feature is coming :)

**Pull request merged**